### ISSUE TITLE
If there a Manual Way to Trigger Could you Doc it in the README?

### ISSUE THREAD

**Issue by LevonBecker** [dolfinus/AutoSitemap#4]
If there is a manual way to trigger and test the sitemap.xml gen with a php command that would be great to have in the README.md.

**Comment by LevonBecker** [comment]
I figured out that making a post/article change it kicked it off. Which was good, I had to adjust my Nginx config to allow access to the the style sheet https://www.whatever.com/extensions/AutoSitemap/sitemap.xsl for the sitemap.xml to display right. I had this (Still learning Nginx conf hackage):

 (This Breaks this Extension)
```nginx 
location ^~ /extensions/ { internal; }
```

(This seems to work)
```nginx
location ^~ /extensions/ { deny all; }
```

I have been fighting with the **internal** setting... doesn't seem to work like it's described.

Anyways, 

Thanks for resurrecting the extension!
